charts/landscaper/templates/service.yaml

```diff
@@ -16,11 +16,23 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
-    {{- if not (has "all" .Values.disableWebhooks) }}
-    - port: {{ .Values.webhookConfig.servicePort }}
-      targetPort: 9443
-      protocol: TCP
-      name: webhook
-    {{- end }}
   selector:
     {{- include "landscaper.selectorLabels" . | nindent 4 }}
+---
+{{- if not (has "all" .Values.disableWebhooks) }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "landscaper.webhooks.fullname" . }}
+  labels:
+  {{- include "landscaper.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  - port: {{ .Values.webhookConfig.servicePort }}
+    targetPort: {{ .Values.webhookConfig.servicePort }}
```

Comment:
I might be confused here, but I think the `targetPort` has to match the webserver's port, which is hardcoded to `9443` [here](https://github.com/schrodit/landscaper/blob/standalone-webhooks/cmd/landscaper-webhooks-server/app/app.go#L57).

Reply:
oh yes good catch. I think I will make it configurable. (the idea is that someone running it locally can run the landsacper controller and the webhook server locally)